pkgs/development/python-modules/gpib-ctypes/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+}:
+
+buildPythonPackage rec {
+  pname = "gpib_ctypes";
+  version = "0.3.0";
+  disabled = pythonOlder "3.7";
```

```suggestion
    version = "0.3.0";
    format = "...."
  
  disabled = pythonOlder "3.7";
```
please add the spacing and set the format, for more information about the format look at the manual in the `buildPythonPackage parameter` section https://nixos.org/manual/nixpkgs/stable/